AirCasting/Utils/Strings.swift

```diff
@@ -1,302 +1,436 @@
-        static let description: String = LocalizedStringKey("Connect AirBeam to measure air quality humidity, and temperature.").stringKey
-        static let sheetButton: String = LocalizedStringKey("Learn More").stringKey
-    }
-
+        static let title: String = NSLocalizedString("Measure and map your exposure to air pollution",
+                                                     comment: "")
+        static let description: String = NSLocalizedString("Connect AirBeam to measure air quality humidity, and temperature.",
+                                                           comment: "")
+        static let sheetButton: String = NSLocalizedString("Learn More",
+                                                           comment: "")
+    }
+    
     enum OnboardingAirBeamSheet {
-        static let sheetTitle: String = LocalizedStringKey("How AirBeam \nworks?").stringKey
-        static let sheetDescription_1: String = LocalizedStringKey("In ").stringKey
-        static let sheetDescription_2: String = LocalizedStringKey("mobile ").stringKey
-        static let sheetDescription_3: String = LocalizedStringKey("mode, the AirBeam \ncaptures personal exposures.\n\nIn ").stringKey
-        static let sheetDescription_4: String = LocalizedStringKey("fixed ").stringKey
-        static let sheetDescription_5: String = LocalizedStringKey("mode, it can be \ninstalled indoors or outdoors to \nkeep tabs on pollution levels in \nyour home, office, backyard, or \nneighborhood 24/7.").stringKey
-    }
-
+        static let sheetTitle: String = NSLocalizedString("How AirBeam works?",
+                                                          comment: "")
+        static let sheetDescription_1: String = NSLocalizedString("In ",
+                                                                  comment: "It consists of few parts. Please consider them together. Whole text is following: In mobile mode, the AirBeam captures personal exposures. In fixed mode, it can be installed indoors or outdoors to keep tabs on pollution levels in your home, office, backyard, or neighborhood 24/7.")
+        static let sheetDescription_2: String = NSLocalizedString("mobile ",
+                                                                  comment: "")
+        static let sheetDescription_3: String = NSLocalizedString("mode, the AirBeam captures personal exposures. In ",
+                                                                  comment: "")
+        static let sheetDescription_4: String = NSLocalizedString("fixed ",
+                                                                  comment: "")
+        static let sheetDescription_5: String = NSLocalizedString("mode, it can be installed indoors or outdoors to keep tabs on pollution levels in your home, office, backyard, or neighborhood 24/7.",
+                                                                  comment: "")
```

Comment:
all this should be a single string too, but to incorporate this into SwiftUI it will be a bit of work, because `Text` doesn't let us style a single string with multiple formats. This split purpose is to distinguish "mobile" and "fixed" in this text. Knowing that we can detect those words in text in our view and split it or style it there. So steps to fix that:
1. Merge this string into one in the Strings file
2. Make sure in strings file we provide localized strings for "mobile" and "fixed" session types
3. Inside the view detect those words, split the string there and concat multiple `Text` accordingly